src/Engine/Contract/Never.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
This isn't really an interface, so there's probably a better place this could go...